crates/bevy_reflect/src/reflect.rs

```diff
@@ -369,29 +383,116 @@ pub trait Reflect: DynamicTypePath + Any + Send + Sync {
     }
 }
 
-impl Debug for dyn Reflect {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        self.debug(f)
+/// A core trait of [`bevy_reflect`], used for downcasting to concrete types.
+///
+/// This is a subtrait of [`PartialReflect`],
+/// meaning any type which implements `Reflect` implements `PartialReflect` by definition.
+///
+/// It's recommended to use [the derive macro] rather than manually implementing this trait.
+/// Doing so will automatically implement this trait, [`PartialReflect`] and many other useful traits for reflection,
```

Comment:
Nit: I believe we include the Oxford comma
```suggestion
/// Doing so will automatically implement this trait, [`PartialReflect`], and many other useful traits for reflection,
```